src/components/cards/BiobankCard.vue

```diff
@@ -244,4 +269,18 @@ export default {
   right: -7rem;
   left: -0.5rem;
 }
+
+.name {
```

Comment:
try to avoid floats, this has side-effects. Better to use flex (see bootstrap utilities)

Reply:
https://getbootstrap.com/docs/4.0/utilities